server/router.js

```diff
@@ -31,7 +32,12 @@ router.on('/:archiveKey', {
   get: function (req, res, params) {
     let state = getDefaultAppState()
     state.archive.key = params.archiveKey
-    sendSPA('/:archiveKey', res, state)
+    var archive = haus(state.archive.key)
```

Comment:
if we're doing `const` we can probably do `let` vs `var`. But in this case I think `const` is legit?